packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.jsx

```diff
@@ -95,7 +97,7 @@ export const IdvDocSubmitOnSignup = ({ citizen_data, has_previous, onPrevious, o
             errors.document_number =
                 localize('Please enter your document number. ') + getExampleFormat(document_type.example_format);
         } else if (
-            (validation_is_enabled && (is_recurring_number || is_sequential_number)) ||
+            (!is_idv_whitelist_document_number && (is_recurring_number || is_sequential_number)) ||
```

Comment:
As also mentioned in the other idv file. I suggest wrapping all these statements in one variable and pass that variable in the `else if` statement